frontend/src/Editor/AppVersionsManager.jsx

```diff
@@ -175,7 +175,7 @@ export const AppVersionsManager = function AppVersionsManager({
                           disabled={isDeletingVersion}
                           style={{
                             display: mouseHoveredOnVersion === version.id ? 'block' : 'none',
-                            marginTop: '3px',
+                            // marginTop: '3px',
```

Comment:
If not required, please remove the line instead of commenting.

Reply:
Made the required change